skill-tree/pom.xml

```diff
@@ -151,6 +151,43 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>2.43.0</version>
+                <configuration>
+                    <java>
+                        <includes>
+                            <include>src/main/java/**/*.java</include>
+                            <include>src/test/java/**/*.java</include>
+                        </includes>
+                        <googleJavaFormat>
+                            <style>GOOGLE</style>
+                            <formatJavadoc>false</formatJavadoc>
+                        </googleJavaFormat>
+                        <indent>
+                            <tabs>true</tabs>
+                            <spacesPerTab>2</spacesPerTab>
+                        </indent>
+                        <indent>
+                            <spaces>true</spaces>
+                            <spacesPerTab>4</spacesPerTab>
```

Comment:
This part is two times here, may be because of some reason. Can you please elaborate once?

Reply:
Hi @satyam73 
This is done to enforce 4 spaces indentation when using tabs. Since, we are using googleJavaFormat, we need to turn google's 2-space indentation to tabs, then the tabs to 4-space indentation. Hence, its written 2 times here.
For more information, the following link can be referred.
https://github.com/diffplug/spotless/issues/420#issuecomment-512350756